frontend/src/components/header/private-elements/private-elements.tsx

```diff
@@ -0,0 +1,53 @@
+import React, { useState } from 'react';
+
+import ArrowDown from '@assets/images/header/arrow-drop-down.svg';
+import DefaultAvatar from '@assets/images/header/default-avatar.png';
+
+import { IRemindersProps, Reminders } from '../reminders/reminders';
+import styles from './private-elements.module.scss';
+
+interface IPrivateComponentProps extends IRemindersProps {
+  avatar: string | undefined | null;
+}
```

Comment:
Please don't use `I` in front of the interfaces. It's not considered as the best practice in TS